src/services/known_tokens.ts

```diff
@@ -0,0 +1,44 @@
+import { WETH_TOKEN_SYMBOL } from '../common/constants';
+import { KNOWN_TOKENS_META_DATA } from '../common/tokens_meta_data';
+import {
+    getWethTokenFromTokensMetaDataByNetworkId,
+    mapTokensMetaDataToTokenByNetworkId,
+} from '../util/token_meta_data';
+import { Token } from '../util/types';
+
+export class KnownTokens {
```

Comment:
Some thoughts on this.

First, can we move this to the `util` directory, based on what we talk about having pure/testable things there?

Second, it would be nice to make `KNOWN_TOKENS_META_DATA` a constructor parameters, so that this can be tested.

Reply:
Yup agree with both items. Will work on that now.